### ISSUE TITLE
Remove topic from consumer group

### ISSUE THREAD

**Issue by cykl** [deviceinsight/kafkactl#88]
When an application is refactored to replace one input topic by another one, old topic must be removed from the consumer group (we track consumer group lag using something similar to https://github.com/lightbend/kafka-lag-exporter).

kafka-consumer-groups allows such operation with a combination of --delete-offsets and --topic.
 
```
./kafka-consumer-groups.sh  --bootstrap-server 10.10.10.10:9092 --delete-offsets --group my_group --topic old_topic
```

kafkactl doesn't seems to support this feature. It is a deliberate choice or a feature gap?

**Comment by random-dwi** [comment]
Hey @cykl,

this is would indeed be a good feature for kafkactl. We are using `sarama` library for kafka communication and there is currently an open issue for this: https://github.com/Shopify/sarama/issues/1912

So one first needs to contribute the related API calls to sarama and then we can implement it.
Any help on this is greatly appreciated.

**Comment by faillefer** [comment]
A pull request for Shopify/sarama#1912 has been merged in sarama's master branch
I will work on supporting this feature in kafkactl
Can we agree on the command definition and options here ?

I think of two possibilites :

1.  Adding a `delete consumer-group-offset` command with mandatory options `--topic`, `--partition` and `--consumer-group`
```sh
# Exemple 
kafkactl delete --consumer-group=my-group --topic=my-topic  --partition=0
```
2.  Adding `--offset` to the existing `delete consumer-group` (only the offset will be deleted, not the consumer-group)
```sh
# Exemple 
kafkactl delete consumer-group my-group --offset --topic=my-topic  --partition=0
```
Any idea on the best options ?

**Comment by random-dwi** [comment]
I would align it with [reset consumer-group offset](https://github.com/deviceinsight/kafkactl/blob/master/cmd/reset/reset-consumer-group-offset.go) and also use the same aliases but under `delete` parent command. So this would then look like:

```
kafkactl delete consumer-group-offset my-group --topic my-topic --partition 12
kafkactl delete cgo my-group --topic my-topic --partition 12
kafkactl delete offset my-group --topic my-topic --partition 12
```
where partition is optional and the default is to delete offsets for all partitions of the topic.

What do you think @faillefer?

**Comment by faillefer** [comment]
@random-dwi  I agree, it's better to align with `reset consumer-group`.
I'll start to work on that next week

**Comment by faillefer** [comment]
Implemented in #101
PR is still WIP still this new feature will only be available in the next release of sarama.

**Comment by random-dwi** [comment]
@faillefer Thank you for the PR. I roughly overlooked the changes and I'm quite happy with the PR :+1: 
Since I currently have limited time I will postpone a more thorough review to when the next sarama version is released.
I hope thats fine for you.

**Comment by faillefer** [comment]
@random-dwi it's fine for me 😉. I wasn't expecting a review before the next Sarama release.

**Comment by random-dwi** [comment]
is available in kafkactl 1.21.0
